src/engine.rs

```diff
@@ -1006,10 +998,11 @@ fn general_count_literal<F: FnMut(u32, u32) -> bool>(drive: &mut WrapDrive, end:
     }
 }
 
-fn eq_loc_ignore(code: u32, ch: u32) -> bool {
-    code == ch || code == lower_locate(ch) || code == upper_locate(ch)
+extern "C" {
+    fn tolower(c: i32) -> i32;
+    fn toupper(c: i32) -> i32;
+    fn isalnum(c: i32) -> i32;
 }
```

Comment:
I don't think we should use the C functions here -- I don't think we really need to care about locales, and this adds an unnecessary dependency on `libc`. I think for the functions that take a u32, you can just use `std::char::from_u32()` instead of `u8::try_from`